spec/support/test_helpers/teacher_training_public_api_helper.rb

```diff
@@ -24,6 +24,16 @@ def stub_teacher_training_api_providers(recruitment_cycle_year: RecruitmentCycle
     )
   end
 
+  def stub_teacher_training_api_providers_for_pagination(recruitment_cycle_year: RecruitmentCycle.current_year)
```

Comment:
```suggestion
def stub_teacher_training_api_providers_for_pagination(recruitment_cycle_year: RecruitmentCycle.current_year)
    page_numbers = [1,2,3]
  
    page_numbers.each do |page_number|
      stub_pagination_request(recruitment_cycle_year, page_number, pagination_response(page_number))
    end
 end
```
Could be simplified further to this but unsure how clear it would be. Thoughts?